SPECS/dracut/20overlayfs/overlayfs-mount.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/sh
+
+# Description: This script is designed to mount a DM-Verity root filesystem and
+# set up an OverlayFS. It is driven by kernel parameters and is invoked during
+# the dracut initramfs phase.
+
+# Kernel Parameters:
+# - root: Specifies the path to the root filesystem. This script expects it to
+#   be a DM-Verity protected device, typically for systemd at
```

Comment:
Curious, why does the script expect that? For example, if I am not using DM-Verity, can I still use the script?

Reply:
Thanks George, yes you can. I will update it. This was a legacy comment message, now I have revised the commit and support the general usage.